cm/example_empty/README.md

```diff
@@ -0,0 +1,7 @@
+# Remarks
+
+The *example_empty* is empty CM.
+
+It only serves as an example for patners developing others CM.
```

Comment:
The *example_empty" is the generic CM structure to be implemented.

Reply:
I made the suggested changes